src/core/digest.js

```diff
@@ -24,11 +24,9 @@
  *
  * @author Bertrand Chevrier <bertrand@taotesting.com>
  */
-import 'webcrypto-shim';
-import { TextEncoder } from 'fastestsmallesttextencoderdecoder';
```

Comment:
These are out because
- [window.crypto.subtle](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/subtle#browser_compatibility) became supported in all browsers and Node
- [TextEncoder](https://developer.mozilla.org/en-US/docs/Web/API/TextEncoder#browser_compatibility) the same